### PR TITLE
sdk: rename call to argument

### PR DESCRIPTION
**Pull request by firedpeanut**
refer to
https://github.com/linera-io/linera-protocol/blob/5101a4851df00d3dad1c86cef939dda4e5b9c1ca/linera-sdk/src/lib.rs#L174

### CODE REVIEW

File: linera-sdk/src/lib.rs [hunk 1]
```diff
@@ -224,16 +224,16 @@ pub trait Contract: WithContractAbi + ContractAbi + Send + Sized {
         &mut self,
         authenticated: bool,
         application: ApplicationId<A>,
-        call: &A::ApplicationCall,
```
Comment (ma2bd) [inline linera-sdk/src/lib.rs:227]: This is intended: `ApplicationCall` -> `call`

Comment (ma2bd) [reply linera-sdk/src/lib.rs:227]: Can you change `argument: Self::ApplicationCall` into `call: Self::ApplicationCall` ? Thanks!

**Review by ma2bd — CHANGES_REQUESTED** [review]
back to your queue for the comments

**Comment by firedpeanut** [comment]
I see.close it

**Pull request closed without merging**